packages/connect/src/types/settings.ts

```diff
@@ -17,7 +17,7 @@ export interface ConnectSettings {
     popup?: boolean;
     transportReconnect?: boolean;
     webusb?: boolean; // deprecated
-    transports?: Transport['name'][];
+    transports?: (Transport['name'] | Transport)[];
```

Comment:
Dunno, this seems a bit strange. Why is there a reason to pass either strings or instances?